### DISCUSSION TITLE
Questions about MLX and Keras.

### DISCUSSION THREAD

**Discussion by Redempt1onzzZZ** [ml-explore/mlx#1155] [General]
I would like to ask if MLX can be used as the backend of Keras like TensorFlow and CNTK？

**Comment by awni** [comment]
There is a WIP on the Keras side to add it as a back-end: https://github.com/keras-team/keras/issues/19571

And the corresponding Keras branch: https://github.com/keras-team/keras/tree/mlx

**Reply by Redempt1onzzZZ** [reply]
Thanks for response! understood.
